platform/darwin/src/MGLAccountManager_Private.h

```diff
@@ -8,4 +8,7 @@
 /// The current global access token.
 @property (atomic) NSString *accessToken;
 
+/// The API base URL that is used to access Mapbox resources. The default base URL is `https://api.mapbox.com`. If `nil`, the Mapbox default base API URL is in use.
+@property (atomic, readwrite) NSURL *apiBaseURL;
```

Comment:
Following up on https://github.com/mapbox/mapbox-gl-native/pull/13754#discussion_r251172506, this should also be `nonatomic`.

Reply:
Why a property that requires thread safety should be marked as `nonatomic`?

Reply:
@fabian-guerra Given that these are set once/rarely and then observed thereafter by KVO, it seems like that should  avoid a case where a different threads might receive differing/incomplete/bad values (as presumably KVO waits until the setter finishes before notifying).

But I see your point and agree they might as well be `atomic`, as the relative speed of setting/getting doesn’t matter for properties that aren’t very active. @jmkiley let’s make these `atomic` across the board.